ocs_ci/helpers/helpers.py

```diff
@@ -1729,6 +1763,93 @@ def verify_volume_deleted_in_backend(
         return False
 
 
+def delete_volume_in_backend(img_uuid, pool_name=None):
+    """
+    Delete an Image/Subvolume in the backend
+
+    Args:
+         img_uuid (str): Part of VolID which represents corresponding
+            image/subvolume in backend, eg:
+            ``oc get pv/<volumeName> -o jsonpath='{.spec.csi.volumeHandle}'``
+            Output is the CSI generated VolID and looks like:
+            ``0001-000c-rook-cluster-0000000000000001-f301898c-a192-11e9-852a-1eeeb6975c91``
+            where image_uuid is ``f301898c-a192-11e9-852a-1eeeb6975c91``
+         pool_name (str): The of the pool
+
+    Returns:
+         bool: True if image deleted successfully
+            False if:
+                Pool not found
+                image not found
+                image not deleted
+
+    """
+    cmd = ""
+    valid_error = []
+    pool_cr = get_pool_cr(pool_name)
+    if pool_cr is not None:
+        if pool_cr["kind"] == "CephFilesystem":
+            interface = "CephFileSystem"
+        else:
+            interface = pool_cr["kind"]
```

Comment:
why dont remove the <if  else> and just use : interface = pool_cr["kind]  ?
it is look the same to me

Reply:
It a weird thing. I query kind `CephFileSystem` and the return dict has `CephFilesystem` with a lower s in system word.
And then query in other functions fail because constants.CEPHFILESYSTEM is with capital S.
Maybe it should be reported.